admin/src/app/Components/registration/registration.component.ts

```diff
@@ -52,9 +52,9 @@ export class RegistrationComponent{
     );
   }
 
-  public register(email: string, password: string ): void {
-    this.authService
-      .register(email, password)
+  public createAdmin(email: string, password: string ): void {
+    this.userService
+      .createUser(email, password)
```

Comment:
Why 4 different names for one operation with one resource? 
registrationForm - createAdmin - userService - accounts